examples/se_2_3_localization.cpp

```diff
@@ -86,15 +136,15 @@
  *  All these variables are summarized again as follows
  *
  *    X   : robot's extended pose, SE_2(3)
- *    u   : robot control input, (R_{k}.T v dt + 0.5*dt*dt*acc_{k}, omega_{k} dt, acc_{k} dt)
- *    Q   : control perturbation covariance
+ *    u   : robot control input, (Jinv * (R.T v dt + 0.5 dt^2 (alpha + R.T g)), omega dt, Jinv * ((alpha + R.T g) dt))
+ *    U   : control perturbation covariance
  *    b_k : k-th landmark position, R^3
  *    y   : Cartesian landmark measurement in robot frame, R^3
  *    R   : covariance of the measurement noise
  *
  *  The motion and measurement models are
  *
- *    X_(t+1) = f(X_t, u) = X_t * Exp ( w )     // motion equation
+ *    X_(t+1) = f(X_t, u) = X_t * Exp ( u )     // motion equation
  *    [y_k; 1; 0]     = h(X, b_k) = X^-1 * [b_k; 1; 0]          // measurement equation
```

Comment:
We must define the action of SE_3_2. But if this action is on points of R3, then the measurements are points of R3, not of R5:

 b \in R3

 y = X^-1 * b \in R3

mind that in SE3 the action is also on R3: X*b = R*b+T , that is, it is not a homogeneous product but the equivalent Euclidean operation.

Reply:
Fixed in https://github.com/artivis/manif/pull/154/commits/ded121b75f0f401c518336cf0d31ad862be50275